PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx

```diff
@@ -566,7 +569,14 @@ Bool_t AliAnalysisTaskNewJetSubstructure::FillHistograms() {
 	{
 	  Double_t fRPAngle = ((AliVAODHeader*)InputEvent()->GetHeader())->GetEventplane();
 	  Double_t phiBinT = RelativePhi(jet1->Phi(),fRPAngle);
-	  fShapesVar[20] = phiBinT;
+	  AliEventplane *aliEP = InputEvent()->GetEventplane();
+	  Double_t fEPV0 = 0;
+	  Double_t phiBinT2 = 0;
+	  if (aliEP) {
+	    fEPV0  = aliEP->GetEventplane("V0" ,InputEvent());
+	    phiBinT2 = RelativePhi(jet1->Phi(),fEPV0);
+	  }
+	  fShapesVar[20] = phiBinT2;
```

Comment:
```suggestion
	  fShapesVar[20] = RelativePhi(jet1->Phi(),fEPV0);
```

AliAnalysisTaskEmcal already retrieves `fEPV0` for you, so you just need this one line